vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -147,9 +148,14 @@ vic_cesm_put_data()
 
                 // albedo: direct , visible
                 // CESM units: unitless
-                albedo = AreaFactor *
-                         (atmos->shortwave[NR] /
-                          (atmos->shortwave[NR] - energy.NetShortAtmos));
+                if (atmos->shortwave[NR] > 0.) {
+                    albedo = AreaFactor *
+                             (atmos->shortwave[NR] /
+                              (atmos->shortwave[NR] - energy.NetShortAtmos));
+                }
+                else {
+                    albedo = 0.;
```

Comment:
@apcraig - I think we discussed this in #256.  What do you think about returning an albedo of zero during dark hours.